### PR TITLE
polypane: init at 10.0.1

### PR DESCRIPTION
**Pull request by zoedsoupe**
###### Motivation for this change
Polypane is a stand-alone browser that target web developers providing a rich support for responsive and acessible web developing tools.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    inherit pname src version;
+  };
+in appimageTools.wrapType2 {
+  inherit pname src version;
+
+  multiPkgs = null;
+  extraPkgs = pkgs: appimageTools.defaultFhsEnvArgs.multiPkgs pkgs ++ [ pkgs.bash ];
+
+  extraInstallCommands = ''
+    ln -s $out/bin/${name} $out/bin/${pname}
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:24]:
```suggestion
    ln -s $out/bin/${pname}-${version} $out/bin/${pname}
```

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{ lib, fetchurl, appimageTools, pkgs }:
+
+let
+  pname = "polypane";
+  version = "8.0.0";
+  name = "${pname}-${version}";
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:6]:
```suggestion
```

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{ lib, fetchurl, appimageTools, pkgs }:
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:1]:
```suggestion
{ lib, fetchurl, appimageTools }:
```

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    install -m 444 -D ${appimageContents}/${pname}.png \
+      $out/share/icons/hicolor/512x512/apps/${pname}.png
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace 'Exec=AppRun' 'Exec=${pname}'
+  '';
+
+  meta = with lib; {
+    description = "Browser with unified devtools targeting responsability and acessibility";
+    longDescription = ''
+    The stand-alone browser for ambitious developers that want to build responsive, accessible and performant websites in a fraction of the time it takes with other browsers.
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:34]:
```suggestion
      The stand-alone browser for ambitious developers that want to build responsive, accessible and performant websites in a fraction of the time it takes with other browsers.
```

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    '';
+    homepage = "https://polypane.app/";
+    maintainers = with maintainers; [ zoedsoupe ];
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:37]:
```suggestion
    maintainers = with maintainers; [ zoedsoupe ];
```

File: maintainers/maintainer-list.nix [hunk 6]
```diff
@@ -7523,16 +7523,6 @@
     githubId = 29855073;
     name = "Michael Colicchia";
   };
-  matdsoupe = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:7526]: You also need to update https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/earthly/default.nix#L40

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    platforms = [ "x86_64-linux" ];
+    changelog = "https://polypane.app/docs/changelog/";
+    license = with licenses.unfree;
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:40]:
```suggestion
    license = licenses.unfree;
```´
this should fix the syntax errror.

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by SuperSandro2000** [comment]
@zoedsoupe can you please do your commits like it is described in the contributing document? If so ofborg will also trigger the build.

@ofborg build polypane

**Comment by zoedsoupe** [comment]
> @zoedsoupe can you please do your commits like it is described in the contributing document? If so ofborg will also trigger the build.
> 
> @ofborg build polypane

Of course!!

File: maintainers/maintainer-list.nix (hunk 6 above)
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:7526]: Please put this into a separate commit with the message ``maintainers: update NAME`` and sort it alphabetically.

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+  src = fetchurl {
+    url = "https://github.com/firstversionist/${pname}/releases/download/v${version}/${pname}-${version}.AppImage";
+    name = "${pname}-${version}.AppImage";
+    sha256 = "eujv99L5svMhDIKHFOfm7sOwNZ4xiUaIsimfOf4BBik=";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit pname src version;
+  };
+in appimageTools.wrapType2 {
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:16]: I also wouldn't mind if that would be rec but not required...

File: pkgs/applications/networking/browsers/polypane/default.nix [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+  multiPkgs = null;
+  extraPkgs = pkgs: appimageTools.defaultFhsEnvArgs.multiPkgs pkgs ++ [ pkgs.bash ];
+
+  extraInstallCommands = ''
+    ln -s $out/bin/${pname}-${version} $out/bin/${pname}
+    install -m 444 -D ${appimageContents}/${pname}.desktop $out/share/applications/${pname}.desktop
+    install -m 444 -D ${appimageContents}/${pname}.png \
+      $out/share/icons/hicolor/512x512/apps/${pname}.png
+    substituteInPlace $out/share/applications/${pname}.desktop \
+      --replace 'Exec=AppRun' 'Exec=${pname}'
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/browsers/polypane/default.nix:28]: ..and we wouldn't substitute pname wherever possible.

File: maintainers/maintainer-list.nix [hunk 10]
```diff
@@ -15175,6 +15175,7 @@
     name = "Zoey de Souza Pessanha";
     email = "zoey.spessanha@outlook.com";
     keys = [{
+      longkeyid = "rsa4096/0x1E1E889CDBD6A315";
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:15178]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**